CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ find_program(SAIL_BIN "sail")
 if (NOT SAIL_BIN)
     message(FATAL_ERROR "Sail not found. See README.md for installation instructions.")
 endif()
+message(STATUS "Found sail: ${SAIL_BIN}")
```

Comment:
Strictly speaking this is unrelated, but I found it helpful considering we print the path to GMP and libz as well.